manifest.json

```diff
@@ -20,18 +20,20 @@
   "oauth_config": {
     "scopes": {
       "bot": [
+        "app_mentions:read",
```

Comment:
note: This is the only new scope. The rest are just ordered alphabetically.

Reply:
⭐ praise: Thank you for finding these additional improvements!